CalibrationWidgets/measureDistBetweenMotors.py

```diff
@@ -128,7 +132,9 @@ def on_Exit(self):
         
         '''
         
-        print "measure dist on exit ran"
         
+        #Restore original chain over sprocket direction
         App.get_running_app().data.config.set('Advanced Settings', 'chainOverSprocket', self.originalChainOverSproketDir)
+        #restore all settings to the values stored in the current settings file 
+        self.data.gcode_queue.put("$$ ")
```

Comment:
Do we feel good about me using $$ to reset all the settings to the values stored in settings? It seems to work great